params/extras/network_upgrades.go

```diff
@@ -264,6 +264,10 @@ type AvalancheRules struct {
 	IsGranite                                                                           bool
 }
 
+func (a AvalancheRules) IsGraniteActivated() bool {
```

Comment:
can we add a comment why this is required (to make it clear this is called from precompile environment etc)